examples/gsxChatCompletion/gensx.yaml

```diff
@@ -0,0 +1 @@
+projectName: derek-sandbox
```

Comment:
I think we should get rid of `gensx.yaml` for now, bring it back when we are ready to talk about this stuff.

Reply:
yep, good call. same with the workflow.tsx. I had removed them at one point but accidentally added them back in  lol